lib/v2/bilibili/followers.js

```diff
@@ -1,11 +1,18 @@
 const got = require('@/utils/got');
 const cache = require('./cache');
+const config = require('@/config').value;
 
 module.exports = async (ctx) => {
     const uid = ctx.params.uid;
+    const login_uid = ctx.params.login_uid;
 
     const name = await cache.getUsernameFromUID(ctx, uid);
 
+    const cookie = config.bilibili.cookies[login_uid];
+    if (cookie === undefined) {
+        throw Error('缺少对应 login_uid 的 Bilibili 用户登录后的 Cookie 值');
+    }
```

Comment:
This check should be placed before declaring `uid` and `login_uid`.
You can add a link to the docs like below as well.
https://github.com/DIYgod/RSSHub/blob/e52c82971fec25a8dec8ec5652f00a8d0df28143/lib/v2/github/notifications.js#L8-L10